sampler/README.md

```diff
@@ -437,17 +439,17 @@ with our sampling function, then they should approximate the exact PDF
 or PMF. If after many samples the approximation is poor or obviously
 wrong, then we know there is a bug in our code somewhere.
 
-Another reason to implement the PMF or PDF is because frequently, you
+Another reason to implement the PMF or PDF is that frequently, you
```

Comment:
"the reason is because" isn't strictly incorrect, but it's inelegant enough to change in this context. (It would be fine in a less formal context.)